admin/database/database.go

```diff
@@ -47,8 +47,8 @@ type DB interface {
 	Migrate(ctx context.Context) error
 	FindMigrationVersion(ctx context.Context) (int, error)
 
-	FindOrganizations(ctx context.Context) ([]*Organization, error)
-	FindOrganizationsForUser(ctx context.Context, userID string) ([]*Organization, error)
+	FindOrganizations(ctx context.Context, paginationOpts *PaginationOptions) ([]*Organization, error)
```

Comment:
Replace the `PaginationOptions` with the actual query parameters – in this case something like `limit int, afterName string`. 

The `database` functions are all very close to the actual query parameters, and having a generic name like `Cursor` makes it easy to accidentally pass a value that doesn't match the underlying column name (like an ID instead of a name).